tests/autoyast/verify_imported_users.pm

```diff
@@ -0,0 +1,31 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Verify that users are imported and not created when a system is re-installed with autoyast
+# by checking for non-existence of .bashrc in /var/lib/{gdm,empty,polkit,nobody,pulseaudio}.
+# Maintainer: Joaquín Rivera <jeriveramoya@suse.com>
+
+use strict;
+use warnings;
+use base 'consoletest';
+use testapi;
+use scheduler 'get_test_data';
+
+sub run {
+    my $test_data = get_test_data();
+    my $errors;
+    foreach my $path (@{$test_data->{paths}}) {
+        $errors .= "$path should not exist\n" if (!script_run("test -f $path"));
+    }
+    die "Bash profiles created instead of being imported:\n $errors" if ($errors);
```

Comment:
Do you think adding reference to the bug is a good idea?

Reply:
Yes, I had it in some point, but I was doing to much Bash before this version and I forgot adding it back. thx.